From ce71feccf991a66677b41fc9a4299dd304781603 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 9 Feb 2018 15:18:21 +0800
Subject: [PATCH] arm64: rockchip_defconfig: remove
 NETFILTER_XT_MATCH_QUOTA2_LOG

bc86c1de1ded ("ANDROID: android-base.cfg: remove NETFILTER_XT_MATCH_QUOTA2_LOG")

There are currently a couple different implementations for this
functionality. Until things are unified, remove the requirement
for this kernel config.

Change-Id: Ibbe93ae71d4b7512be397e689abc36ba51011b30
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index bc86b44ad0e5..fecd026a5a3f 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -163,7 +163,6 @@ CONFIG_NETFILTER_XT_MATCH_PKTTYPE=y
 CONFIG_NETFILTER_XT_MATCH_QTAGUID=y
 CONFIG_NETFILTER_XT_MATCH_QUOTA=y
 CONFIG_NETFILTER_XT_MATCH_QUOTA2=y
-CONFIG_NETFILTER_XT_MATCH_QUOTA2_LOG=y
 CONFIG_NETFILTER_XT_MATCH_SOCKET=y
 CONFIG_NETFILTER_XT_MATCH_STATE=y
 CONFIG_NETFILTER_XT_MATCH_STATISTIC=y
-- 
2.35.3

